tmt/cli.py

```diff
@@ -2034,7 +2034,7 @@ def setup_completion(shell: str, install: bool) -> None:
 
     if install:
         with open(script, 'w') as f:
-            subprocess.run(command, stdout=f)
+            subprocess.run(command, stdout=f, check=False)
```

Comment:
why adding `check=False` here?

Reply:
If I remember correctly, it was about explicitly setting it instead of relying on default value, but now when I remove it, nothing complains. So I guess it is indeed redundant.

Reply:
I would keep the original code myself, but no strong preference :)

Reply:
> I would keep the original code myself, but no strong preference :)

Do you mean the Path.write_text I've changed this to in the last commit, or just removing the `check=False`?